Makefile

```diff
@@ -43,7 +42,6 @@ format: ## Format the source code
 
 tidy: ## Update dependencies
 	$(Q)go mod tidy
-	$(Q)go mod vendor
```

Comment:
I added this in #262 but I realize we had already removed the vendor directory for this repository so it's unneeded now.